web/package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "@thorvg/lottie-player",
+  "version": "1.0.0-beta.0",
+  "description": "A web lottie player using ThorVG as a renderer",
+  "main": "./dist/lottie-player.cjs.js",
+  "module": "./dist/lottie-player.esm.js",
+  "exports": {
+    ".": {
+      "import": "./dist/lottie-player.cjs.js",
+      "require": "./dist/lottie-player.esm.js"
+    }
+  },
+  "types": "./dist/lottie-player.d.ts",
+  "scripts": {
+    "build": "npm run parse:version && npm run clean && npm run copy:wasm && rollup -c --bundleConfigAsCjs && rm -rf ./dist/thorvg-wasm.js",
+    "build:watch": "npm run clean && npm run copy:wasm && rollup -c --bundleConfigAsCjs --watch",
+    "copy:wasm": "cp ../build_wasm/src/bindings/wasm/thorvg-wasm.js ./dist",
+    "clean": "rm -rf dist && mkdir dist && touch dist/index.js",
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "lint": "eslint ./src --ext .ts,.tsx,.js",
+    "lint:fix": "eslint ./src --ext .ts,.tsx,.js --fix",
+    "parse:version": "export THORVG_VERSION=`sed -n -e 4p ../meson.build | sed 's/..$//' | sed -r 's/.{19}//'`"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/thorvg/thorvg.git"
+  },
+  "author": "Jinny You <jinny@lottiefiles.com>",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/thorvg/thorvg/issues"
+  },
+  "homepage": "https://thorvg.org",
+  "keywords": [
+    "lottie",
+    "animation",
+    "thorvg",
+    "tvg"
```

Comment:
We're not officially supporting `.dotlottie` in ThorVG.

Reply:
the "keywords" prop in `package.json` helps people discover your package as it's listed in 'npm search'. I thought it would be useful since the latest dotlottie Lottie player based on ThorVG. But yeah, up to you